pages/ksphere/konvoy/1.5/release-notes/index.md

```diff
@@ -315,1693 +315,6 @@ enterprise: false
 -   traefik-forward-auth: 1.0.4-7
 -   velero: 1.0.1-5
 
-### Version v1.4.5 - Released 26 June 2020
```

Comment:
We can have a convo about this but maybe only have minor release info for each version of RN.

Reply:
@dkoshkin